src/main/java/edu/harvard/iq/dataverse/api/Index.java

```diff
@@ -489,7 +489,7 @@ public String getSolrSchema() {
             sb.append("   <field name=\"" + nameSearchable + "\" type=\"" + type + "\" multiValued=\"" + multivalued + "\" stored=\"true\" indexed=\"true\"/>\n");
         }
 
-        List<String> listOfStaticFields = new ArrayList();
+        List<String> listOfStaticFields = new ArrayList<>();
```

Comment:
@bencomp you're certainly right that this is a type safety issue but the default settings from Netbeans apparently don't provide developers the typical yellow warning indicators:

![screen shot 2015-09-25 at 9 35 47 am](https://cloud.githubusercontent.com/assets/21006/10101853/2a816dba-6369-11e5-9cb6-f0bde0020573.png)

If you could please share the Netbeans settings you are using, I'd appreciate it. You could even add it to the Google Doc linked from http://guides.dataverse.org/en/latest/developers/coding-style.html